android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -69,6 +71,7 @@
 import com.reactnativecommunity.webview.events.TopLoadingStartEvent;
 import com.reactnativecommunity.webview.events.TopMessageEvent;
 import com.reactnativecommunity.webview.events.TopShouldStartLoadWithRequestEvent;
+import com.reactnativecommunity.webview.jsi.Lifecycle;
```

Comment:
Perhaps Lifecycle isn't the best name for this class. It doesn't involve any kinds of setup or teardown methods that you would think of as actual lifecycle methods. Perhaps JsiInterface, JsiBridge, WebviewSynchronousJsiInterface, WebViewJsiInterface, or something of that ilk would be a better name.